src/dotnet/CarbonAware/CarbonAwareCore.cs

```diff
@@ -1,24 +1,29 @@
 ﻿namespace CarbonAware;
 
 using CarbonAware.Interfaces;
+using Microsoft.Extensions.Logging;
 
 /// <summary>
 /// Carbon Aware SDK Core class, called via CLI, native, and web endpoints.
 /// </summary>
 public class CarbonAwareCore : ICarbonAwareBase
 {
     private readonly ICarbonAwarePlugin _plugin;
+    private readonly ILogger<CarbonAwareCore> _logger;
 
-    public CarbonAwareCore(ICarbonAwarePlugin plugin)
+
+    public CarbonAwareCore(ILogger<CarbonAwareCore> logger, ICarbonAwarePlugin plugin)
     {
-        _plugin = plugin;
-
-        //Console.WriteLine($"Carbon Aware Core loaded with carbon logic.");
-        //Console.WriteLine($"\tName: '{plugin.Name}'");
-        //Console.WriteLine($"\tAuthor: '{plugin.Author}'");
-        //Console.WriteLine($"\tDescription: '{plugin.Description}'");
-        //Console.WriteLine($"\tVersion: '{plugin.Version}'");
-        //Console.WriteLine($"\tURL: '{plugin.URL}'");
+        this._plugin = plugin;
+        this._logger = logger;
+
+        _logger.LogInformation("Carbon Aware Core loaded with carbon logic.");
+
+        //_logger.LogDebug($"\tName: '{plugin.Name}'");
```

Comment:
I would remove these commented out lines. We don't need them OR we could mark them as verbose logs

Reply:
I've marked them as Debug, since dotnet doesn't have a "verbose" by default. I had intended to bring this up and ask whether we wanted these logs at all, but I think we should keep them for now and remove if needed.